src/config.rs

```diff
@@ -54,6 +54,8 @@ pub struct FrontendConfig {
 pub struct DatabaseConfig {
     /// The database connection URL.
     pub url: String,
+    /// Max connection pool size
+    pub connection_pool_max_size: Option<u32>,
```

Comment:
I would suggest having a simpler name for this configuration option, something like:
```suggestion
    pub max_conns: Option<u32>,
```

Reply:
Good call! Committing..